rpc/handle_get_connected_peer_info.go

```diff
@@ -0,0 +1,22 @@
+package rpc
+
+import (
+	"github.com/kaspanet/kaspad/rpc/model"
+)
+
+// handleGetConnectedPeerInfo implements the getConnectedPeerInfo command.
+func handleGetConnectedPeerInfo(s *Server, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	peers := s.protocolManager.Peers()
+	infos := make([]*model.GetConnectedPeerInfoResult, 0, len(peers))
+	for _, peer := range peers {
+		info := &model.GetConnectedPeerInfoResult{
```

Comment:
Please create ticket if there are any fields that are left TODO.

Reply:
Done.